tests/map_test.cpp

```diff
@@ -29,10 +29,10 @@ TEST_CASE( "map_coordinate_conversion_functions" )
         here.vertical_shift( 0 );
     } );
 
-    tripoint_bub_ms test_point =
-        GENERATE( tripoint_bub_ms::zero, tripoint_bub_ms::zero + tripoint::south,
-                  tripoint_bub_ms::zero + tripoint::east, tripoint_bub_ms::zero + tripoint::above,
-                  tripoint_bub_ms::zero + tripoint::below );
+    tripoint test_point =
+        GENERATE( tripoint::zero, tripoint::south,
+                  tripoint::east, tripoint::above,
+                  tripoint::below );
```

Comment:
```suggestion
    tripoint_bub_ms test_point = tripoint_bub_ms::zero +
        GENERATE( tripoint_rel_ms::zero, tripoint_rel_ms::south,
                  tripoint_rel_ms::east, tripoint_rel_ms::above,
                  tripoint_rel_ms::below );
```
If your PR works as is (ie this line is the problem) I think the above should work, I think the funky GENERATE define ( [here](https://github.com/CleverRaven/Cataclysm-DDA/blob/c766338599840c45b4d4ae48a0d5fbd870ae4118/tests/catch/catch.hpp#L4096) ) just doesn't like you passing unresolved args. We already have a test generating tripoint_rel_ms here https://github.com/CleverRaven/Cataclysm-DDA/blob/c766338599840c45b4d4ae48a0d5fbd870ae4118/tests/char_sight_test.cpp#L95 so it should work fine.

Reply:
This is a variant of what @mqrause suggested, and I have no problems with it (apart from possible test failure, but at least this will test that). I've modified the code accordingly, but also removed the now rather useless next line and its usage to clean up the code a little. That's why the suggested change wasn't just accepted directly.

Reply:
Ye I didn't look at the other PR until after commenting here '^^

Reply:
Two great minds thinking along the same line indicates it's probably correct...